models/sessions/scratch/bigquery/snowplow_web_sessions_this_run.sql

```diff
@@ -0,0 +1,303 @@
+{{
+    config(
+        tags=["this_run"]
+    )
+}}
+
+with my_firsts as (
+    select
+        -- app id
+        app_id as app_id,
+
+        -- session fields
+        domain_sessionid,
+        domain_sessionidx,
+
+        {{ snowplow_utils.current_timestamp_in_utc() }} as model_tstamp,
+
+        -- user fields
+        user_id,
+        domain_userid,
+        {% if var('snowplow__session_stitching') %}
+            -- updated with mapping as part of post hook on derived sessions table
+            cast(domain_userid as {{ snowplow_utils.type_string(255) }}) as stitched_user_id,
+        {% else %}
+            cast(null as {{ snowplow_utils.type_string(255) }}) as stitched_user_id,
+        {% endif %}
+        network_userid as network_userid,
+
+        -- first page fields
+        page_title as first_page_title,
+        page_url as first_page_url,
+        page_urlscheme as first_page_urlscheme,
+        page_urlhost as first_page_urlhost,
+        page_urlpath as first_page_urlpath,
+        page_urlquery as first_page_urlquery,
+        page_urlfragment as first_page_urlfragment,
+
+        -- referrer fields
```

Comment:
I'm sure you've considered this and it might be too much of a change, but what about prepending the following fields with `first_`?

Reply:
We could do, the reason I kept it like this was just to align to the existing structure of the table - it means it's not a breaking change from a structure point of view and is instead just "breaking" in that the logic has changed. I can change it if we want to though?